docs/perf_test_result_dumps.md

```diff
@@ -149,3 +149,39 @@ Command as above
   %{blknum: 5000, span_ms: 4059, tps: 2931, txs: 11897}
 ]
 ```
+
+## `8240f8b2c828c30853b6a34d8fa1b89b75da399e`, 2019-09-24
+
+run extended perftest with 35 sender (opts: %{randomized:false}), dev.period 3 seconds
+
+```
+[
+  %{blknum: 1000, span_ms: 611, tps: 3905, txs: 2386},
+  %{blknum: 2000, span_ms: 5307, tps: 4107, txs: 21798},
+  %{blknum: 3000, span_ms: 6000, tps: 3962, txs: 23771},
+  %{blknum: 4000, span_ms: 6030, tps: 3674, txs: 22156},
+  %{blknum: 5000, span_ms: 6014, tps: 3460, txs: 20809},
+  %{blknum: 6000, span_ms: 2431, tps: 4352, txs: 10580}
+]
+```
+
+then pick one of senders at random, call
```

Comment:
Let's have a single narrative of the example of the perf test invocation. Now we have three, each slightly different (here, demo_05 and in byzantine_events.ex).

Can we make it more consistent?

Maybe add a top-level function that allows you to kick off a parametrized test of this kind in a one-liner. Then you'd fire this function here, in the smoke test and in the docs? Not 100% sure though